index.js

```diff
@@ -185,6 +185,12 @@ async function main() {
     }
   })
 
+  app.post('/api/whoami', async (request, response) => {
+    const { username } = await getUserBySessionID(request.body.sessionID)
+
+    response.end(JSON.stringify({ username }))
+  })
+
```

Comment:
Is this used in your PR?

It looks like you use it to get the name of the user from a session, which you use for highlighting mentions. You should use the new session API for this - or, better yet, just merge with my commits and use `(sessionObj && sessionObj.user.username === 'whatever')`.